packages/framework/src/hooks/useRegisterBindings.ts

```diff
@@ -58,7 +65,7 @@ export const useRegisterBindings = <T extends { [key: string]: unknown }>(
       (options?.comparator
         ? options.comparator(newValues, widgetState?.values)
         : isEqual(newValues, widgetState?.values)) &&
-      isEqual(keys(methods), keys(widgetState?.invokable.methods))
+      lodashIsEqual(keys(methods), keys(widgetState?.invokable.methods))
```

Comment:
Can we leave a comment here explaining we need this to compare function equality?